### PR TITLE
changes in getTokens()

### PR DESCRIPTION
**Pull request by Marielbamar21**
### Changes
An array was created with all the information of all the tokens and the getTokens() was modified to return the data from the new array.

### Comment
please, to check.

### CODE REVIEW

File: src/stealthex/stealhtex.interface.ts [hunk 1]
```diff
@@ -3,20 +3,20 @@ export interface StealthExApiError {
...
+//   name: string
+//   warnings_from: string[]
+//   warnings_to: string[]
+//   validation_address: string
+//   validation_extra: string
+//   address_explorer: string
+//   tx_explorer: string
+//   image: string
+//   network: string
+// }
```
Comment (0xslipk) [inline src/stealthex/stealhtex.interface.ts]: Remove this code, if not longer need it

File: src/stealthex/stealhtex.service.ts [hunk 2]
```diff
@@ -52,18 +51,17 @@ export class StealthExService {
...
-  public async getTokens(): Promise<StealthExToken[]> {
-    try {
-      const { data } = await this.sendPetition<StealthExToken[]>('currency', 'get')
-
-      return data
-    } catch (err) {
-      this.logger.error({ err }, 'Error getting tokens.')
+  //     return data
+  //   } catch (err) {
+  //     this.logger.error({ err }, 'Error getting tokens.')
```
Comment (0xslipk) [inline src/stealthex/stealhtex.service.ts]: same!

**Review by rubenguc — APPROVED** [review]
LGTM

**Review by 0xslipk — APPROVED** [review]
LGTM

**Pull request merged**